### DISCUSSION TITLE
weitere Produkte hinzufügen

### DISCUSSION THREAD

**Discussion by Ben040Zin** [Fetty42/FS22_VIPOrderManager#16] [Ideas]
Hey. 

Wäre es möglich, weitere Produkte  wie z.B. Mehl oder Öl zu den Aufträgen hinzuzufügen.
Das würde die Aufgaben noch etwas vielfältiger machen.

Gruß.
Ben

**Comment by Fetty42** [comment]
Hi,
eigentlich werden auch die nicht in der "VIPOrderManagerDefaults.lua" definierten FillTypes verwendet. Hierfür wird der ftConfig "DEFAULT_FILLTYPE" verwendet. Also erst ab Level 6 mit einer geringen Wahrscheinlichkeit.

**Reply by Ben040Zin** [reply]
Dann hatte ich wohl bislang einfach keine solche Aufgaben erhalten. Lieben Dank für die Antwort.
